### PR TITLE
update some top-level markdown files

### PR DESCRIPTION
**Pull request by ebuchman**
Still need to fix up the CONTRIBUTING.md a bit more (add notes on how changelog should look)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ please [contact us](mailto:partners@tendermint.com) and [join the chat](https://
 ## Security
 
 To report a security vulnerability, see our [bug bounty
-program](https://tendermint.com/security).
+program](https://https://hackerone.com/tendermint)
```
[the remark this answers is not in the document]
Comment (melekes) [reply README.md]: good candidate for Github suggestion feature

[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: i cant seem to activate it :(

[the remark this answers is not in the document]
Comment (melekes) [reply README.md]:
```suggestion
program](https://hackerone.com/tendermint)
```

[the remark this answers is not in the document]
Comment (melekes) [reply README.md]: @zramsay

[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]:
```suggestion
program](https://hackerone.com/tendermint)
```

[the remark this answers is not in the document]
Comment (zramsay) [reply README.md]: ohhh, this will be a huge time-saver

File: CHANGELOG_PENDING.md [hunk 2]
```diff
@@ -2,24 +2,12 @@
 
 ## v0.26.2
 
-*TBA*
+*November 15th, 2018*
```
[the remark this answers is not in the document]
Comment (ebuchman) [reply CHANGELOG_PENDING.md]: Hoping to set a release today but you're right

**Pull request merged**